encrypt_data.py

```
@@ -0,0 +1,112 @@
+import django
+
+# Set up Django environment
+os.environ.setdefault("DJANGO_SETTINGS_MODULE", "web.settings")
+django.setup()
+
+# Now import models and crypto functions
+from web.crypto_utils import encrypt_value  # noqa: E402
+from web.models import Donation, FeatureVote, PeerMessage, Profile, WebRequest  # noqa: E402
+
+
+def encrypt_existing_data():
```

Ruff (0.8.2)

`encrypt_existing_data` is too complex (21 > 10)

(C901)